core/src/main/com/github/mmorihiro/core/ArrayActor.kt

```diff
@@ -0,0 +1,33 @@
+package com.github.mmorihiro.core
+
+import com.badlogic.gdx.graphics.g2d.Batch
+import com.badlogic.gdx.graphics.glutils.ShapeRenderer
+import com.badlogic.gdx.scenes.scene2d.Actor
+
+
+class ArrayActor : Actor() {
+    private val arraySize = 20
+    private var array = Array(arraySize) { BooleanArray(arraySize) { false } }
```

Comment:
Oh, I noticed this variable should be `val`.
Array is imutable by default in Scala, but is mutable in Kotlin.